src/afl-fuzz-bitmap.c

```diff
@@ -474,7 +474,8 @@ save_if_interesting(afl_state_t *afl, void *mem, u32 len, u8 fault) {
 
   /* Generating a hash on every input is super expensive. Bad idea and should
      only be used for special schedules */
-  if (unlikely(afl->schedule >= FAST && afl->schedule <= RARE)) {
+  if (unlikely((afl->schedule >= FAST && afl->schedule <= RARE) ||
+               afl->coverage_estimation)) {
```

Comment:
can you please change this to:

`if (likely(afl->schedule >= FAST && afl->schedule <= RARE) || unlikely(afl->coverage_estimation))`

Reply:
Are you sure this would change the effect? Since the unlikely was already there. I can change it to likely, but then I would change the meaning of the line. I haven't changed the unlikely.
Do you mean:
`if (unlikely(afl->schedule >= FAST && afl->schedule <= RARE) || unlikely(afl->coverage_estimation))`
I could change that, but I don't know if this would differ to the current state.

Reply:
no I meant what I wrote :)  the fast schedule is the default so it makes more sense it is likely()